pbs_nodefile.c

```diff
@@ -0,0 +1,162 @@
+/*
+ * Plugin to set the PBS_NODEFILE env variable and its contents for the job
+ *
+ * (C) 2018 Ghent University
+ *
+ * Author: Andy Georges
+ *
+ * Loosely based on Spank plugin private-tmpdir (c) HPC2N.umu.se
+ *
+ * This plugin generates the PBS_NODEFILE in the $TMPDIR of the job. We assume
+ * other plugins or job epilog will clean these up, so we do not remove files
+ * ourselves.
```

Comment:
add a TODO or know issues wrt the double file

Reply:
Done.